packages/portal/backend/test/CSVParserSpec.ts

```diff
@@ -2,11 +2,11 @@ import {expect} from "chai";
 import "mocha";
 
 import Log from "../../../common/Log";
+import {Test} from '../../../common/TestHarness';
```

Comment:
I'm really surprised these moves worked; I'm sure I tried putting `Test` in `common` (where it totally should be) and for some reason it wasn't working for me. Maybe the new version of TS handles imports better or something. Anyways, glad to see this in there!

Reply:
@rtholmes 

I did not run into any versioning issues with TypeScript while moving the `Test` class into the `common` area. There is a compilation issue to be aware of, however, which is that compiling AutoTest in the Docker container requires that the `backend` portal application is also pulled into the AutoTest container. It would be nice if we could somehow change this compilation dependency to allow these applications to compile separately, but I have not come across any viable solutions as of yet.

Reply:
Ahh, I guess this is the problem I ran into. I'll also have to check to make sure the tests can still be run interactively in Webstorm, since this remains the best way to debug test failures.

Reply:
That's a good point. I have been using VSCode to run the test suites, so it would be important to try tests in Webstorm before making changes to the test suite setups. 

The package.json test commands certainly continue to work.

Reply:
@rtholmes did you want to make sure Webstorm works first before merging this PR?

Reply:
No, if it's working in VSCode we can just deploy the change and revert out of it later if we need to.